chatlearn/utils/vllm_utils.py

```diff
@@ -389,6 +401,10 @@ def __init__(self, src_names, layer_offset, qwen_version=QwenVersion.v_1):
             "modules": ["mlp.w1", "mlp.w2"],
             "dim": 0
         }
+        self._to_fix_shared_expert_ordering = {
+            "modules": ["mlp.shared_experts.dense_h_to_4"],
```

Comment:
dense_h_to_4h?

Reply:
BTW, existing tests didn't reveal the typo here.

Reply:
In QWen-moe model, "dense_h_to_4h" appeas in two weights, thus the whole name should be specified.